LICENSE.txt

```diff
@@ -21,3 +21,10 @@ DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
 OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
 NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
 EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+------------------------------------------------------------------------------
+
+FFmpeg that is redistributed within all PyAV wheels and binaries is licensed
+under GNU General Public License (GPL) version 2 or later. If those binaries
```

Comment:
How do we know that the GPL V2 is the one that applies as FFmpeg is linked against many libraries with different licenses?

Reply:
I think we really need to look at what other projects distributing wheels with FFmprg are doing. I'm feeling somewhat out of my depth on this topic.